src/js/components/modals/ServiceRestartModal.js

```diff
@@ -21,10 +21,26 @@ class ServiceRestartModal extends ServiceActionModal {
       }
     ];
 
-    this.onMarathonStoreServiceRestartError = this.onError;
     this.onMarathonStoreServiceRestartSuccess = this.closeDialog;
   }
 
+  onMarathonStoreServiceRestartError({message:errorMsg = '', details}) {
+    let hasDetails = details && details.length !== 0;
+
+    if (hasDetails) {
+      this.setState({
+        errorMsg: details.reduce(function (memo, error) {
+
+          return `${memo} ${error.errors.join(' ')}`;
```

Comment:
But it'd be nice to have a check that `errors` is set here, this feels vulnerable to reference error.